ports/psoc6/mpconfigport.h

```diff
@@ -26,21 +26,23 @@
 
 // Options controlling how MicroPython is built, overriding defaults in py/mpconfig.h
 #include <stdint.h>
-#include "mpconfigboard.h"
```

Comment:
Why is this removed? With this include we can add board configuration, and provide here default port configuration, but prevent to overwrite that the board with #ifdef/#ifndef.

Reply:
Also if the flag is coming from the compiler

Reply:
I guess its just moved to the end of the file so that nothing board-specific is overwritten.

Reply:
Right, it has been moved to the end such that board specific changes overwrite port specific ones. Alternatively, all defines have to be enclosed in #ifndef ... #endif, which is much more messy than this way.

Reply:
okay... as long as the defines are properly overwritten by the board configuration.

Reply:
Leaving it this way for now, if decided to to it differently we could change it as we incorporate the WIFI changes.